src/qtAliceVision/SequenceCache.hpp

```diff
@@ -144,6 +148,15 @@ class SequenceCache : public QObject, public ImageServer {
     /// Target size used to compute downscale
     int _targetSize;
 
+    /// Local threadpool
+    QThreadPool _threadPool;
```

Comment:
why not use the global thread pool instance ?

Reply:
using the global thread pool lead to crashes sometimes. I do not have more explanations than this !